### PR TITLE
[10.0] [ADD] product_manufacturer_add_supplierinfo

### PR DESCRIPTION
**Pull request by nikul-serpentcs**
Product manufacturer add supplier info
======================================

Extended product template. This module allows to automatically create a product
supplier from the product manufacturer information.

### CODE REVIEW

**Comment by nikul-serpentcs** [comment]
@aheficent Could you please review.

File: product_manufacturer_add_supplierinfo/__init__.py [hunk 1]
```diff
@@ -1,6 +1,8 @@
 # -*- coding: utf-8 -*-
 # © 2015-17 Eficent Business and IT Consulting Services S.L.
 # - Jordi Ballester Alomar
+# Copyright 2017 Serpent Consulting Services Pvt. Ltd.
```
Comment (AaronHForgeFlow) [inline product_manufacturer_add_supplierinfo/__init__.py:4]: Please reserve this for authors. Contributors who migrate can go in the contributors section in the readme.

**Review by AaronHForgeFlow — CHANGES_REQUESTED** [review]
Functional ok

**Comment by AaronHForgeFlow** [comment]
I'll merge and do the thing.

**Pull request merged**